tests/app/views/test_dev_mode.py

```diff
@@ -43,3 +44,10 @@ def test_dev_flush_mode_submission(self):
                       'collection_exercise_sid': '789',
                       'ru_ref': '12346789012A'})
         self.assertStatusNotFound()
+
+    def test_extract_eq_id_and_form_type_no_form_type(self):
+        #Given
+        data_to_extract = extract_eq_id_and_form_type('rogueone.json')
+
+        self.assertEqual(data_to_extract, ('rogueone', '-1'))
```

Comment:
It's worth reading up about the testing pyramid (e.g. http://www.agilenutshell.com/episodes/41-testing-pyramid and http://www.velocitypartners.net/blog/2014/01/28/agile-testing-the-agile-test-automation-pyramid/

In summary there should be more test coverage at the unit test level than component or UI; as these tests tend to be easier to write, faster to run and less likely to break. Worth chatting about this some more.